device/esp_tinyusb/tinyusb.c

```diff
@@ -66,8 +78,18 @@ esp_err_t tinyusb_driver_install(const tinyusb_config_t *config)
     return ESP_OK;
 }
 
-esp_err_t tinyusb_driver_uninstall()
+esp_err_t tinyusb_driver_uninstall(void)
```

Comment:
Could we call this function from `tearDown()`?
Once a test case fails, the PHY does not get released, thus all the following test cases would fail because of the:

```
E (23549) usb_phy: usb_new_phy(296): selected PHY is in use
E (23559) TinyUSB: tinyusb_driver_install(65): Install USB PHY failed
```